openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/TypeFormatTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *Copyright (c) 2023,WSO2 LLC. (https://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.openapi.generators.schema;
+
+import io.ballerina.compiler.syntax.tree.SyntaxTree;
+import io.ballerina.openapi.core.GeneratorUtils;
+import io.ballerina.openapi.core.exception.BallerinaOpenApiException;
+import io.ballerina.openapi.core.generators.schema.BallerinaTypesGenerator;
+import io.ballerina.openapi.generators.common.TestUtils;
+import io.swagger.v3.oas.models.OpenAPI;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * This includes primitive data types format types tests.
+ */
+public class TypeFormatTests {
+    private static final Path RES_DIR = Paths.get("src/test/resources/generators/schema").toAbsolutePath();
+
+    @Test
+    public void stringFormats() throws IOException, BallerinaOpenApiException {
```

Comment:
Better to stick with the common test method naming conventions.

```suggestion
    public void testStringFormats() throws IOException, BallerinaOpenApiException {
```